CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.1] - 2019-02-25
+
+### Features
```

Comment:
Some folks may consider the addition of a new rule to be somewhat breaking, as if they are letting CI fail on warnings this could break a build which has nothing to do with this.

Three options for a happy life:

1. We introduced it as info in 5.0.1 and warning in 5.1.0
2. Remove from this release entirely and put it in 5.1.0
3. This release is 5.1.0

Don't mind which, but to offer an option: it might be a fairly weak 5.1.0 as is. I would lean towards 1 or 2 but I really don't mind what you decide.

Reply:
Well, I'm not opinionated at all. We in fact have at least a single feature in, small but still a feature, so if we wanted to be compliant with semantic versioning, this should be a 5.1.0.
Let's do it 5.1.0 then.